llvm_util/llvm2alive.cpp

```diff
@@ -915,6 +916,25 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
     case llvm::Intrinsic::prefetch:
       return NOP(i);
 
+    // X86 intrinsics
```

Comment:
can you move this above the NOP ones pls?

Reply:
moved